hub/constraints/instruction-handling/acc.lisp

```diff
@@ -23,21 +23,45 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 
-(defun (account-instruction-requires-trimming)        (+ [ stack/DEC_FLAG 1 ]
-                                                         [ stack/DEC_FLAG 2 ]
-                                                         [ stack/DEC_FLAG 3 ]))
-(defun (account-instruction-no-trimming)              (- stack/ACC_FLAG (account-instruction-requires-trimming)))
-(defun (account-instruction-raw-address-hi)           [ stack/STACK_ITEM_VALUE_HI 1 ])
-(defun (account-instruction-raw-address-lo)           [ stack/STACK_ITEM_VALUE_LO 1 ])
-(defun (account-instruction-account-address-hi)       context/ACCOUNT_ADDRESS_HI)
-(defun (account-instruction-account-address-lo)       context/ACCOUNT_ADDRESS_LO)
-(defun (account-instruction-byte-code-address-hi)     context/BYTE_CODE_ADDRESS_HI)
-(defun (account-instruction-byte-code-address-lo)     context/BYTE_CODE_ADDRESS_LO)
-(defun (account-instruction-address-warmth)           account/WARMTH)
-(defun (account-instruction-decoded-flags-sum)        (+ [ stack/DEC_FLAG 1 ]
-                                                         [ stack/DEC_FLAG 2 ]
-                                                         [ stack/DEC_FLAG 3 ]
-                                                         [ stack/DEC_FLAG 4 ])) ;; ""
+;; ROFF ≡ ROW_OFFSET
+;; ACC  ≡ ACCOUNT_FAMILY
+(defconst
+  ROFF_ACC___CONTEXT_ROW            1
+  ROFF_ACC___ACCOUNT_READING_ROW    2
+
+  ROFF_ACC___ACCOUNT_DOING_ROW      1
+  ROFF_ACC___ACCOUNT_UNDOING_ROW    2
+  )
+
+
+(defun (account-instruction---raw-address-hi)           [ stack/STACK_ITEM_VALUE_HI 1 ])
+(defun (account-instruction---raw-address-lo)           [ stack/STACK_ITEM_VALUE_LO 1 ])
+
+(defun (account-instruction---result-hi)                [ stack/STACK_ITEM_VALUE_HI 4 ])
+(defun (account-instruction---result-lo)                [ stack/STACK_ITEM_VALUE_LO 4 ])
+
+(defun (account-instruction---is-BALANCE)               [ stack/DEC_FLAG 1 ] )
+(defun (account-instruction---is-EXTCODESIZE)           [ stack/DEC_FLAG 2 ] )
+(defun (account-instruction---is-EXTCODEHASH)           [ stack/DEC_FLAG 3 ] )
+(defun (account-instruction---is-CODESIZE)              [ stack/DEC_FLAG 4 ] ) ;; ""
+(defun (account-instruction---is-SELFBALANCE)           (-    1
```

Comment:
for clarity, maybe better (- 1 (+  _sum shorthands)) ? instead of (- 1 \sum shorthands)